tools/common_lib/src/dnnl_utils.h

```diff
@@ -84,6 +85,17 @@ inline dnnl::memory::desc to_dnnl_mem_desc(const TensorShape& shape, const DataL
     return dnnl::memory::desc{ to_dnnl_dims(shape), to_dnnl_data_type(t), to_dnnl_format(l) };
 }
 
+inline dnnl::memory::desc matrix_transpose(const dnnl::memory::desc& nchw_desc, const DataType& t)
```

Comment:
1. This function does not actually do any matrix transpose, it just sets different layout, so the name is/could be confusing. I propose to rename itL: convert_to_ncwh_layout(..) or something similar
2. `const DataType& t` is not used, can be removed from the args. Anyway data type can be queried from `nchw_desc. argument.